src/Json.Schema.ToDotNet.UnitTests/Json.Schema.ToDotNet.UnitTests.csproj

```diff
@@ -13,7 +13,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <ItemGroup>
-    <None Include="TestData\*.schema.json">
+    <Compile Remove="TestData\**\*" />
```

Comment:
>TestData [](start = 21, length = 8)

The test data directory now contains some perfectly good C# files with names like "ExpectedClass.cs". Don't compile them!